src/torcontrol.cpp

```diff
@@ -285,17 +290,61 @@ static std::map<std::string,std::string> ParseTorReplyMapping(const std::string
             ++ptr; // skip opening '"'
             bool escape_next = false;
             while (ptr < s.size() && (escape_next || s[ptr] != '"')) {
```

Comment:
I'm also don't understand why escape_next is not allowed to be true twice in a row (line 294)